build.fsx

```diff
@@ -68,15 +68,15 @@ Target.create "RestoreServer" (fun _ ->
 
 Target.create "Build" (fun _ ->
     runDotNet "build" serverPath
-    runDotNet "fable webpack-cli -- --config src/Client/webpack.config.js -p" clientPath
+    runTool yarnTool "webpack --config src/Client/webpack.production.js" clientPath
 )
 
 Target.create "RunWeb" (fun _ ->
     let server = async {
         runDotNet "watch run" serverPath
     }
     let client = async {
-        runDotNet "fable webpack-dev-server -- --config src/Client/webpack.config.js" clientPath
+        runTool yarnTool "webpack-dev-server --config src/Client/webpack.development.js" clientPath
```

Comment:
So how does it run fable now? (Just to clarify 😄)

Reply:
The eventual F# to JS transpilation is done by the [fable-compiler](https://www.npmjs.com/package/fable-compiler) package which is also added to the dependencies. `fable-compiler` has inside Fable dotnet binaries.